ui/src/timeMachine/components/view_options/CheckOptions.tsx

```diff
@@ -1,49 +0,0 @@
-// Libraries
-import React, {FC} from 'react'
-import {connect} from 'react-redux'
-import {Form, Grid} from '@influxdata/clockface'
-
```

Comment:
Since the `CheckPlot` is only ever used in alerting in the VEO, there is no option to select `CheckOptions`